recipes/libsvm/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "3.21" %}
+{% set flatversion = version.replace(".", "") %}
+
+package:
+  name: libsvm
+  version: {{ version }}
+
+source:
+  fn: libsvm-{{ flatversion }}.tar.gz
+  url: https://github.com/cjlin1/libsvm/archive/v{{ flatversion }}.tar.gz
+  md5: acff2c058b7cb38a135fa5d0b4f39e43
+
+  patches:
+    - win32.patch  # [win32]
+
+build:
+  number: 0
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}                    # [win]
```

Comment:
I agree completely.  I understand it in the build section as necessary to activate features.  It may not be necessary - but that's how I understand it.

The whole scheme of vc features needs an overhaul.